### ISSUE TITLE
Avoid showing cancelled projects everywhere except dashboard

### ISSUE THREAD

**Issue by MoeNick** [Giveth/giveth-next#345]

**Comment by divine-comedian** [comment]
can you clarify how to test this issue?

**Comment by mohammadranjbarz** [comment]
> can you clarify how to test this issue? @mohammadranjbarz

I did nothing for this issue yet, 
@MoeNick can you specify the `everywhere`, I think it can be part of DOD ( because I don't know where we show cancelled projects now)

**Comment by MoeNick** [comment]
If Cancelled (Only by admin) it means no one is able to see it in projects list, search, or even see its detail if the user has the direct url, But Project owner of this project can see it in his list in his account. but cant see the details, can't even edit or share it. 
I now it's both F.E. and Backend, but neeed your help to know do we have any restriction on the backend for such projects , what are our risks?

**Comment by mohammadranjbarz** [comment]
> If Cancelled (Only by admin) it means no one is able to see it in projects list, search, or even see its detail if the user has the direct url, But Project owner of this project can see it in his list in his account. but cant see the details, can't even edit or share it. I now it's both F.E. and Backend, but neeed your help to know do we have any restriction on the backend for such projects , what are our risks?

Thanks, it's almost clear now,
I think we may not have any risk and it can be implemented, but it's good to hear @mateodaza  opinion

**Comment by mateodaza** [comment]
> If Cancelled (Only by admin) it means no one is able to see it in projects list, search, or even see its detail if the user has the direct url, But Project owner of this project can see it in his list in his account. but cant see the details, can't even edit or share it. I now it's both F.E. and Backend, but neeed your help to know do we have any restriction on the backend for such projects , what are our risks?

Yes this is correct. If admin cancelled the project the user can't activate it back, if user deactivated it he's allowed. It's already like this in the backend. Public queries for projects won't show deactivated nor cancelled projects. They will be shown for the owner but he won't be able to change the status.

We can close this unless you see something outstanding @MoeNick
